### PR TITLE
Prepare R for translation

### PR DESCRIPTION
**Pull request by FransMeerhoff**
- abtestbayesian.R Bayesian A/B Test
- summerystatsbinomialtestbayesian. R[Summary Stats] Bayesian Binomial Test

### CODE REVIEW

File: JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R [hunk 1]
```diff
@@ -195,26 +195,26 @@ SummaryStatsBinomialTestBayesian <- function(jaspResults, dataset = NULL, option
...
-    plot$setError("Plotting not possible: Posterior too peaked!")
+    plot$setError(gettext("Plotting not possible: Posterior too peaked!"))
     return()
   }
   
   ppCri           <- c(CIlower, CIupper)
   dfLinesPP       <- .dfLinesPP(dataset = NULL, a = a, b = b, hyp = hypothesis, theta0 = theta0, counts = successes, n = n)
   dfPointsPP      <- .dfPointsPP(dataset = NULL, a = a, b = b, hyp = hypothesis, theta0 = theta0, counts = successes, n = n)
-  xName           <- expression(paste("Population proportion", ~theta))
+  xName           <- expression(paste(gettext("Population proportion"), ~theta))
```
Comment (JorisGoosen) [inline JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R]: expression will make you cry

Comment (JorisGoosen) [reply JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R]: aka gettext must be removed here and later @vandenman  or @TimKDJ can help

Comment (vandenman) [reply JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R]: okay to fix this do:

```r
xName <- bquote(paste(.(gettext("Population proportion")), ~theta))
```
`bquote` is similar to expression but everything inside `.()` is evaluated.

File: JASP-Engine/JASP/R/abtestbayesian.R [hunk 2]
```diff
@@ -163,7 +163,7 @@ ABTestBayesian <- function(jaspResults, dataset, options, ...) {
   if (orNotEqualTo1Prob > 0) {
     rowCount = rowCount + 1
     output.rows[[rowCount]] <- list(
-      "Models"    = "Log odds ratio \u2260 0",
+      "Models"    = gettextf("Log odds ratio %s %s","\u2260", " 0"),
```
Comment (TimKDJ) [inline JASP-Engine/JASP/R/abtestbayesian.R]: `gettextf("Log odds ratio %s 0", "\u2260"),`

File: JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R [hunk 3]
```diff
@@ -242,19 +242,19 @@ SummaryStatsBinomialTestBayesian <- function(jaspResults, dataset = NULL, option
     
     hypothesis_for_common_functions   <- "twoSided"
     hypothesis                        <- "two.sided"
-    message <- paste0("Proportions tested against value: ", theta0, ".")
+    message <- paste0(gettext("Proportions tested against value: "), theta0, ".")
```
Comment (TimKDJ) [inline JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R]: `gettextf("Proportions tested against value: %i.", theta0)`

File: JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R [hunk 4]
```diff
@@ -242,19 +242,19 @@ SummaryStatsBinomialTestBayesian <- function(jaspResults, dataset = NULL, option
...
     
   } else if (hypothesis_option == "greaterThanTestValue") {
     
     hypothesis_for_common_functions   <- "plusSided"
     hypothesis                        <- "greater"
-    message <- paste0("For all tests, the alternative hypothesis specifies that the proportion is greater than ", theta0, ".")
+    message <- paste0(gettext("For all tests, the alternative hypothesis specifies that the proportion is greater than "), theta0, ".")
```
Comment (TimKDJ) [inline JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R]: `gettextf("For all tests, the alternative hypothesis specifies that the proportion is greater than %i.", theta0)`

File: JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R [hunk 5]
```diff
@@ -242,19 +242,19 @@ SummaryStatsBinomialTestBayesian <- function(jaspResults, dataset = NULL, option
...
     
   } else if (hypothesis_option == "lessThanTestValue") {
     
     hypothesis_for_common_functions   <- "minSided"
     hypothesis                        <- "less"
-    message <- paste0("For all tests, the alternative hypothesis specifies that the proportion is less than ", theta0, ".")
+    message <- paste0(gettext("For all tests, the alternative hypothesis specifies that the proportion is less than "), theta0, ".")
```
Comment (TimKDJ) [inline JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R]: `gettextf("For all tests, the alternative hypothesis specifies that the proportion is less than %i.", theta0)`

Comment (FransMeerhoff) [reply JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R]: Done

File: JASP-Engine/JASP/R/abtestbayesian.R (hunk 2 above)
Comment (FransMeerhoff) [reply JASP-Engine/JASP/R/abtestbayesian.R]: Done

File: JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R (hunk 3 above)
Comment (FransMeerhoff) [reply JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R]: done

File: JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R (hunk 4 above)
Comment (FransMeerhoff) [reply JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R]: Done

**Pull request merged**